packages/disputer/index.js

```diff
@@ -178,6 +178,7 @@ async function run({
           message: "End of serverless execution loop - terminating process"
         });
         await waitForLogger(logger);
+        await delay(2); // waitForLogger does not always work 100% correctly in serverless. add a delay to ensure logs are captured upstream.
```

Comment:
Why was this failing before? Because GCP imposes some default timeout?

Reply:
Maybe its worth extending this even longer in case the spokes become more beefy in the future

Reply:
Shold we add this to the `disputerConfig` obj